### ISSUE TITLE
Increase precision of mean depth in output

### ISSUE THREAD

**Issue by eharr** [brentp/mosdepth#26]
This might be a niche use case, but when dealing with low coverage datasets (for instance batches that will subsequently merged) it would be nice to be able to report the mean depth to a higher precision. Ideally this is something that could be specified on the command line.

**Comment by brentp** [comment]
good idea. for the sake of not further complicating the command-line options, I would make this an environment variable like `MOSDEPTH_PRECISION=5`

**Comment by eharr** [comment]
That's perfect - thanks!

**Comment by brentp** [comment]
this is implemented. will be out in next release coming soon.
